src/core/jsonschema/official_resolver.in.cc

```diff
@@ -3,11 +3,15 @@
 auto sourcemeta::core::schema_official_resolver(std::string_view identifier)
     -> std::optional<sourcemeta::core::JSON> {
   // JSON Schema 2020-12
-  if (identifier == "https://json-schema.org/draft/2020-12/schema") {
+  if (identifier == "https://json-schema.org/draft/2020-12/schema" ||
+      // Just for compatibility given that this is such a common issue
+      identifier == "https://json-schema.org/draft/2020-12/schema#") {
     return sourcemeta::core::parse_json(
         R"EOF(@METASCHEMA_JSONSCHEMA_2020_12@)EOF");
   } else if (identifier ==
-             "https://json-schema.org/draft/2020-12/hyper-schema") {
+                 "https://json-schema.org/draft/2020-12/hyper-schema" ||
+             identifier ==
+                 "https://json-schema.org/draft/2020-12/hyper-schema#") {
```

Comment:
Can you add the comment about compatibility above this line too?

Reply:
done